src/pyclaw/solver.py

```diff
@@ -163,7 +178,8 @@ def __init__(self,riemann_solver=None,claw_package=None):
         self._is_set_up = False
         self._use_old_bc_sig = False
         self.accept_step = False
-        self.adjust_dt = False
+        self.get_dt = False
```

Comment:
I don't think you need this variable.  Isn't it just there to indicate whether you are at the first step?  Couldn't you just check whether self.cfl has a meaningful value or not?